src/main/java/com/iexec/sms/tee/session/scone/SconeSessionSecurityConfig.java

```diff
@@ -19,20 +19,22 @@
 import com.iexec.commons.poco.tee.TeeFramework;
 import com.iexec.sms.tee.ConditionalOnTeeFramework;
 import lombok.Getter;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Configuration;
+import lombok.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConstructorBinding;
 
+import javax.validation.constraints.NotNull;
 import java.util.List;
 
-@Configuration
+@Getter
```

Comment:
The `@Getter` annotation is included in the `@Value` annotation
https://projectlombok.org/features/Value
```suggestion
```